graph/db/sql_store.go

```diff
@@ -1267,3 +1464,72 @@ func maybeCreateShellNode(ctx context.Context, db SQLQueries,
 
 	return id, nil
 }
+
+// upsertChanPolicyExtraSignedFields updates the policy's extra signed fields in
+// the database. This includes updating any existing types, inserting any new
+// types, and deleting any types that are no longer present.
+func upsertChanPolicyExtraSignedFields(ctx context.Context, db SQLQueries,
+	chanPolicyID int64, extraFields map[uint64][]byte) error {
+
+	// Get any existing extra signed fields for the channel policy.
```

Comment:
Why can't we just delete all extra types and insert the new ones?
Would probably be more efficient from a SQL standpoint as well: 1 delete query and a couple of inserts.